prisma/schema.prisma

```diff
@@ -55,14 +55,20 @@ model AccountOtp {
   updatedAt             DateTime               @updatedAt
 }
 
+enum sessionType {
```

Comment:
I've added enum and the column to Session model so we can distinguish between users quickly. in the future we might need things as device info, ip address, etc